### PR TITLE
Afsaneh/remove wallet home page

### PR DESCRIPTION
**Pull request by afsaneh-rezaeinejad-deriv**

### CODE REVIEW

File: cypress/e2e/smoke/Wallets/demoResetBalance.cy.js [hunk 1]
```diff
@@ -18,30 +18,26 @@ function reset_balance_demo(platform) {
   cy.findByText('Success').should('exist')
   cy.findByRole('button', { name: 'Transfer funds' }).click()
   //To check if Transfer tab is active on clicking Transfer funds
-  cy.get('[class*="wallets-cashier-header__tab"].wallets-cashier-header__tab')
-    .findByText('Transfer')
-    .parent()
-    .should('be.visible')
-    .invoke('attr', 'class') //would return the string of that class
-    .should('include', 'wallets-cashier-header__tab--active') //find if the class has "active" string
+  cy.findByRole('button', { name: 'Transfer' }).parent().should('be.visible')
```
Comment (afsaneh-rezaeinejad-deriv) [inline cypress/e2e/smoke/Wallets/demoResetBalance.cy.js:21]: this part is validated in demoTransaction test.

File: cypress/e2e/smoke/Wallets/demoResetBalance.cy.js [hunk 2]
```diff
@@ -18,30 +18,26 @@ function reset_balance_demo(platform) {
...
 
   it('should be able to reset balance for demo wallet', () => {
     cy.log('Reset Balance for Demo Account')
-    cy.c_visitResponsive('/wallets', 'large')
+    cy.c_visitResponsive('/', 'large')
     cy.contains('Wallet', { timeout: 10000 }).should('exist')
     reset_balance_demo('desktop')
   })
-  it('should be able to reset balance for demo wallet in responsive', () => {
+  it.only('should be able to reset balance for demo wallet in responsive', () => {
```
Comment (abuzar-deriv) [inline cypress/e2e/smoke/Wallets/demoResetBalance.cy.js:36]: @afsaneh-rezaeinejad-deriv , you want only this test to run? Or this unintentional?

File: cypress/support/commands/common.js [hunk 3]
```diff
@@ -152,15 +135,16 @@ Cypress.Commands.add('c_login', (options = {}) => {
...
+    // } else if (
+    //   (Cypress.env('oAuthUrl') == '<empty>' && app == 'wallets') ||
+    //   app == 'doughflow'
+    // ) {
+    //   getWalletOAuthUrl((oAuthUrl) => {
+    //     cy.log('came inside wallet getOauth')
+    //     Cypress.env('oAuthUrl', oAuthUrl)
+    //     cy.c_doOAuthLogin(app, { rateLimitCheck: rateLimitCheck })
+    //   })
+    // }
```
Comment (abuzar-deriv) [inline cypress/support/commands/common.js:147]: Why comment it out? Why not just remove it if it's not needed?

**Review by abuzar-deriv — APPROVED** [review]
LGTM

**Pull request merged**